hoomd/hpmc/IntegratorHPMC.cc

```diff
@@ -258,6 +258,9 @@ void export_IntegratorHPMC(pybind11::module& m)
         .def("getMPS", &IntegratorHPMC::getMPS)
         .def("getCounters", &IntegratorHPMC::getCounters)
         .def("communicate", &IntegratorHPMC::communicate)
+        .def("getkT", &IntegratorHPMC::getkT)
+        .def("setkT", &IntegratorHPMC::setkT)
+        .def_property("kT", &IntegratorHPMC::getkT, &IntegratorHPMC::setkT)
```

Comment:
```suggestion
        .def_property("kT", &IntegratorHPMC::getKT, &IntegratorHPMC::setKT)
```
We should need only the property.